make.js

```diff
@@ -175,6 +179,78 @@ target.bootlint = () => {
     }, 2000);
 };
 
+target.htmllint = () => {
+    const port = 3081;
+
+    env.PORT = port;
```

Comment:
It's failing CI because the port your assigning isn't being applied on startup. `process.env.PORT` should work though, as the first line of the `start` target is `const env = process.env`.